lightkurve/targetpixelfile.py

```diff
@@ -826,6 +861,64 @@ def get_bkg_lightcurve(self, aperture_mask=None):
                           flux=np.nansum(self.flux_bkg[:, aperture_mask], axis=1),
                           flux_err=self.flux_bkg_err)
 
+    def get_model(self, star_priors=None, **kwargs):
+        """Returns a `TargetPixelFileModel` object with appropriate defaults.
+
+        Parameters
+        ----------
+        star_priors : list of StarPrior objects
+            Stars to include in the model.
+        **kwargs : dict
+            Extra arguments to be passed to the `TPFModel` constructor.
+
+        Returns
+        -------
+        model : TPFModel object
+            Model with appropriate defaults for this Target Pixel File.
+        """
+        from .prf import TPFModel, StarPrior, BackgroundPrior
+        from .prf import GaussianPrior, UniformPrior
+        # Set up the model
+        if star_priors is None:
+            centr_col, centr_row = self.centroids()
+            star_priors = [StarPrior(col=GaussianPrior(mean=np.nanmedian(centr_col), var=0.1**2),
+                                     row=GaussianPrior(mean=np.nanmedian(centr_row), var=0.1**2),
+                                     flux=UniformPrior(lb=0.5*np.nanmax(self.flux[0]),
+                                                       ub=2*np.nansum(self.flux[0]) + 1e-10),
+                                     targetid=self.targetid)]
+        background_prior = BackgroundPrior(flux=GaussianPrior(mean=0.,
+                                           var=(2 * np.nanstd(self.flux_bkg))**2))
+        model = TPFModel(star_priors=star_priors,
+                         background_prior=background_prior,
+                         prfmodel=self.get_prf_model(),
+                         fit_background=True,
+                         **kwargs)
+        return model
+
+    def prf_photometry(self, cadences=None, use_multiprocessing=True, **kwargs):
+        """Returns the results of PRF photometry applied to the pixel file.
+
+        Parameters
+        ----------
+        cadences : list of int
+            Cadences to fit.  This can be useful while debugging a subset of
+            lightcurve cadences.
```

Comment:
I was thinking whether ``fit_parallel`` would fit better than ``use_multiprocessing``.
In any case, this is more of a reminder to add docstring for this kw argument.

Reply:
Let's explain what happens when ``cadences is None`` here, as well?

Reply:
Yes, good idea!